frontend/src/themes/components/button.ts

```diff
@@ -27,8 +29,70 @@ const Button = {
       borderRadius: "16px",
       fontWeight: "700",
     },
+
+    PaginationStyle: {
+      _active: {
+        borderRadius: "50%",
+      },
+      height: "2.25rem",
+      width: "2.25rem",
+      borderRadius: "50%",
+      textAlign: "center",
+      lineHeight: "2rem",
+      display: "inline-block",
+      margin: "0 2px",
```

Comment:
```suggestion
      height: "2rem",
      width: "2rem",
      borderRadius: "50%",
      margin: "0 2px",
      fontWeight: "700",
      fontSize: "12px",
```
generally, you want to only add styling if it's not there by default. also, make sure the styles are consistent with Figma